tests/postgres/migrations/test_legacy_schema_migration.py

```diff
@@ -202,74 +201,5 @@ def mock_migrate(**kwargs):
     return test_migrate_alias_internal
 
 
-def test_migrate_index_versioning(monkeypatch):
```

Comment:
i deleted this test because the order things are initialized and tables are created/migrated has changed, and to make this test pass we would need to mock/override too many things in the test and we would not be testing the real logic